superchain/types.go

```diff
@@ -34,13 +34,10 @@ type ChainConfig struct {
 	Addresses AddressesConfig `toml:"addresses"`
 }
 
-type Dependency struct {
-	ChainIndex     uint32 `json:"chainIndex" toml:"chain_index"`
-	ActivationTime uint64 `json:"activationTime" toml:"activation_time"`
-}
+type StaticConfigDependency struct{}
```

Comment:
Why the rename? In the op-geth superchain package, we don't care about interfaces and static implementations like in the op-supervisor. So I think the simpler `Dependency` would be better, unless there's another reason to call it that.

Reply:
Was aligning it with the name used in op-supervisor. Will change it back

Reply:
Reverted the change here https://github.com/ethereum-optimism/op-geth/pull/619/commits/06f1b219896b78f41cbb192bbea1aa5da206d1ed